tiny_mce/tinymce-4.9.11/js/tinymce/tinymce.js

```diff
@@ -14178,7 +14178,9 @@
           return;
         }
         if (!/^(mceAddUndoLevel|mceEndUndoLevel|mceBeginUndoLevel|mceRepaint)$/.test(command) && (!args || !args.skip_focus)) {
-          editor.focus();
+          setTimeout(function () {
```

Comment:
do we need to worry about clearing the timer?

Reply:
This timer will be trigger when user click on Bold, Italic toolbar option on tinymce editor. I don't think it will create any memory leakage issue here.